AIFG-3D-Connect-4/FourTechRulesHandler.cpp

```diff
@@ -54,8 +34,8 @@ void FourTechRulesHandler::update()
 	else if (m_piecesPlaced == m_TOTAL_BOARD_TILES)
 		m_onGameOverFunction(PieceType::None);
 
-	// Switch turns.
-	m_playersTurn = !m_playersTurn;
+	// Change the turn order
```

Comment:
Misleading comment, the order of the turns is not changing, just the turn itself.